packages/x-date-pickers-pro/src/managers/useDateRangeManager.ts

```diff
@@ -35,7 +35,10 @@ export function useDateRangeManager<TEnableAccessibleFieldDOMStructure extends b
         ...getDateFieldInternalPropsDefaults({ defaultDates, utils, internalProps }),
       }),
       // TODO v8: Add a real aria label before moving the opening logic to the field on range pickers.
-      internal_getOpenPickerButtonAriaLabel: () => '',
+      internal_getOpenPickerButtonAriaLabel: ({ value, utils, localeText }) => {
```

Comment:
Nitpick: I would keep this for a separate PR and introduce new labels for range Pickers to avoid this strange behavior. 🙈 
WDYT?

Reply:
Sure, no problem then, we can adjust afterwards. 👌